templates/vue/src/components/NotificationBar.vue

```diff
@@ -0,0 +1,227 @@
+<template>
```

Comment:
Since this is only for reviews, should we name it something like `ReviewNotifications` or something instead? Also, after the component re-organization is done, this should probably be grouped in a folder with the toolbar.